tests/Get-DbaDbRole.Tests.ps1

```diff
@@ -0,0 +1,52 @@
+﻿$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "Get-DbaDbRole Unit Tests" -Tag "UnitTests" {
+	InModuleScope dbatools {
+		Context "Validate parameters" {
+			$params = (Get-ChildItem function:\Get-DbaDbRole).Parameters
+			it "should have a parameter named SqlInstance" {
+				$params.ContainsKey("SqlInstance") | Should Be $true
+			}
+			it "should have a parameter named SqlCredential" {
+				$params.ContainsKey("SqlCredential") | Should Be $true
+			}
+			it "should have a parameter named Database" {
+				$params.ContainsKey("Database") | Should Be $true
+			}
+			it "should have a parameter named ExcludeDatabase" {
+				$params.ContainsKey("ExcludeDatabase") | Should Be $true
+			}
+			it "should have a parameter named EnableException" {
+				$params.ContainsKey("EnableException") | Should Be $true
+			}
```

Comment:
If you check the templates for test we have shortened testing for the parameters, it runs a little more efficiently as well.

Test I did for `Get-DbaServerRole` is [here](https://github.com/wsmelton/dbatools/blob/2e12188c888a9f86fdab1cf050da017e4d221857/tests/Get-DbaServerRole.Tests.ps1) if you want to see real example.

Reply:
ha, I was looking for that.
thanks